### PR TITLE
Fixes emote cooldown

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Removes all remnants of the broken SR emote cooldown, reverting back to standard TG.

## Why It's Good For The Game

Cooldown on emotes work properly, people can flip again (without beyblading.)

## Proof Of Testing
<details>
<summary>Screenshots/Videos</summary>

https://github.com/user-attachments/assets/8cbda9aa-ba78-4082-a620-d4ac93f8df65

</details>

## Changelog

:cl: LT3
fix: Emote cooldowns work properly
del: Removed Freerunning restriction on the flip emote
/:cl:

### CODE REVIEW

**Review by Majkl-J — APPROVED** [review]
I love you

File: code/modules/mob/emote.dm [hunk 1]
```diff
@@ -82,10 +81,19 @@
 	mob_type_allowed_typecache = list(/mob/living, /mob/dead/observer, /mob/camera/imaginary_friend)
 	mob_type_ignore_stat_typecache = list(/mob/dead/observer, /mob/living/silicon/ai, /mob/camera/imaginary_friend)
 
+// BUBBER EDIT CHANGE BEGIN - Flip Cooldown
 /datum/emote/flip/run_emote(mob/user, params , type_override, intentional)
 	. = ..()
+	var/mob/living/carbon/flippy_mcgee = user
+	if(flippy_mcgee)
```
Comment (thegrb93) [inline code/modules/mob/emote.dm:88]: Casting a different type won't make it null, it'll just runtime later. A byondism I've learned the hard way

File: code/modules/mob/emote.dm [hunk 2]
```diff
@@ -107,6 +115,30 @@
...
 				span_notice("You stumble a bit from still being off balance from your last flip.")
 			)
+*/
+/datum/emote/flip/check_cooldown(mob/living/carbon/user, intentional)
+	. = ..()
+	if(.)
+		return
+	if(!can_run_emote(user, intentional=intentional))
+		return
+	if(!iscarbon(user))
```
Comment (thegrb93) [inline code/modules/mob/emote.dm:125]: I think the input arg needs to be mob/living/user, otherwise this if check always passes

**Comment by thegrb93** [comment]
Love this. I noticed someone commented a bug they found on my PR. I added comments here for the bugs.

**Comment by thegrb93** [comment]
Yeah, I couldn't reproduce the vomit. Weird

**Comment by NargaInDisguise** [comment]
IT COULD NOT be more fucking over...

**Pull request merged**